ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/copy_to_clipboard.js

```diff
@@ -0,0 +1,137 @@
+const copyStyle = `
+.code-n-btn-wrapper {
+  positon: relative;
+}
+.copy-code-wrapper {
+  display: none;
+  position: absolute;
+  right: 0;
+  margin: 10px 15px 0 0;
+}
+.copy-code-wrapper button {
+   background: #fff;
+   border: none;
+   padding: 10px;
+   cursor: pointer;
+}
+.copy-code-wrapper button i {
+font-size: 1rem;
+}
+.code-n-btn-wrapper:hover .copy-code-wrapper{
+  display: block;
+}
+.u-mdic-copy-notify {
+  display: none;
+  padding: 4px 12px;
+  background: #fff;
+  color: #ccc;
+  font-size:0.9rem;
+  margin: 0 10px;
+  border-radius: 10px;
+}
+`;
+
+function generateUuid() {
+	return `${+Date.now()}-${parseInt(Math.random() * 100000)}`;
+}
+
+function strEncode(str = '') {
+	if (!str || str.length === 0) {
+		return '';
+	}
+	return str
+		.replace(/</g, '&lt;')
+		.replace(/>/g, '&gt;')
+		.replace(/'/g, '&apos;')
+		.replace(/"/g, '&quot;');
+}
```

Comment:
Encoding/decoding doesn't work as expected. For example, I have this in my code block: `<b>M&amp;M's</b>` and press Copy, I want to get that exact text back, with no encoding/decoding. But with your current implementation I would get this back `<b>M&M's</b>`

Reply:
I think I’ll have to remove this entire function because I found out markdown-it already handle encoding and decoding